src/App.jsx

```diff
@@ -7,18 +7,28 @@ import About from './pages/About';
 import Contact from './pages/Contact';
 import './App.css';
 
-const App = () => {
+function App() {
   return (
     <Router>
       <Header />
       <Routes>
-        <Route path="/" element={<Home />} />
-        <Route path="/menu" element={<Menu />} />
+        <Route
+          path="/" element={
+            <div>
+              <section id="home" className="section home">
+                <Home />
+              </section>
+              <section id="menu" className="section menu">
+                <Menu />
+              </section>
+            </div>
```

Comment:
these two components are required for the proper functionality of the sections in the current implementation. This will be updated and optimized in a future refactor.